src/components/pages/Order/Main/AdminPanel/AdminPanel.js

```diff
@@ -0,0 +1,93 @@
+import React, { useContext, useState } from "react"
+import styled from "styled-components"
+import AdminContext from "../../../../../context/AdminContext"
+import PanelTabButton from "./PanelTabButton"
+import { FaPen, FaPlus, FaChevronDown, FaChevronUp } from "react-icons/fa"
+import { theme } from "../../../../../theme"
+import PanelOption from "./Panel"
+
+const { colors, fonts, spacing } = theme
+
+function AdminPanel() {
+   const { isActiveTab, setIsActiveTab, isPanelVisible, setIsPanelVisible } =
+      useContext(AdminContext)
+   const [panelOption, setPanelOption] = useState("Ajouter un produit")
+
+   function displayPanel() {
+      setIsPanelVisible(!isPanelVisible)
+   }
+
+   function displayTab() {
+      setIsPanelVisible(true)
+      setIsActiveTab(!isActiveTab)
+      setPanelOption(
+         !isActiveTab ? "Ajouter un produit" : "Modifier un produit"
+      )
+   }
+
+   return (
+      <AdminPanelStyled>
+         <div className="admin-panel-options">
+            <PanelTabButton
+               className={!isPanelVisible && "isActive"}
+               onClick={displayPanel}
+               icon={
+                  isPanelVisible ? (
+                     <FaChevronDown className="icon" />
+                  ) : (
+                     <FaChevronUp className="icon" />
+                  )
+               }
+            />
+            <PanelTabButton
+               className={isActiveTab && " isActive optionTab"}
+               onClick={!isActiveTab ? displayTab : null}
```

Comment:
```suggestion
               onClick={!isActiveTab && displayTab}
```

comme on en parlait l'autre jour 😉

Reply:
Pour celui-là je l'ai redéfini comme ça car sinon il me génère une erreur dans la console comme quoi le onClick attend une fonction et qu'à la place il se retrouve avec un boolean(false quand je switch sur le mode admin)